src/components/venueSignin.js

```diff
@@ -27,49 +26,49 @@ export default class SignIn extends Component {
      this.setVenue(res);
    })
   }
-  
+
   validSignIn(res){
-    if( res != undefined){
+    if( res !== undefined){
       this.setVenue(res[0])
     }
     else{
       //error
     }
   }
+
    setVenue = (res) => {
-    this.setState({
-      venueSignedIn : res[0]
-    })
-    console.log(this.state.venueSignedIn)
+    this.props.action(res);
```

Comment:
hey can we rename action to something more descriptive?

Reply:
Yes! Also realised that I left the argument name in the parent as 'res' and that should be renamed to something better too - will do tomorrow